server/ast/column_table_def.go

```diff
@@ -65,6 +65,10 @@ func nodeColumnTableDef(node *tree.ColumnTableDef) (_ *vitess.ColumnDefinition,
 	if err != nil {
 		return nil, err
 	}
+	// Wrap any default expression in parens to match MySQL's column default requirements
+	if defaultExpr != nil {
+		defaultExpr = &vitess.ParenExpr{Expr: defaultExpr}
```

Comment:
We could check if it's already wrapped so we're not getting double parenthesis.

Reply:
Good call. Blindly wrapping every default expression in parens caused some other tests to fail, so it's now limited to only when the default expression is a function call.